src/test/resources/emissary/test/core/schemas/answers.xsd

```diff
@@ -0,0 +1,586 @@
+                <xs:element ref="fileType"/>
+                <xs:element ref="meta"/>
+                <xs:element ref="altView"/>
+                <xs:element ref="inputAlternateView"/>
+                <xs:element ref="badAlternateView"/>
+            </xs:choice>
+        </xs:complexType>
+    </xs:element>
+    <xs:element name="answers" type="ibdo"/>
+    <xs:element name="att" type="ibdo"/>
+    <xs:element name="extract" type="ibdo"/>
+    <xs:element name="initialForm" type="baseType"/>
+    <xs:element name="fileType" type="baseType"/>
+    <xs:element name="currentFormSize" type="baseType"/>
+    <xs:element name="fontEncoding" type="baseType"/>
+    <xs:element name="inputAlternateView" type="baseType"/>
+    <xs:element name="badAlternateView" type="baseType"/>
+    <xs:element name="broken" type="baseType"/>
+    <xs:element name="procError" type="baseType"/>
+    <xs:element name="extractCount" type="baseType"/>
+    <xs:element name="classification" type="baseType"/>
+    <xs:element name="dataLength" type="baseType"/>
+    <xs:element name="shortName" type="baseType"/>
+    <xs:element name="numAttachments" type="baseType"/>
+    <xs:element name="name" type="xs:string"/>
+    <xs:element name="value" type="xs:string"/>
+    <xs:element name="currentForm">
+        <xs:complexType mixed="true">
+            <xs:complexContent mixed="true">
+                <xs:extension base="baseType">
+                    <xs:attribute name="index" type="xs:integer"/>
+                </xs:extension>
+            </xs:complexContent>
+        </xs:complexType>
+    </xs:element>
+    <xs:element name="altView">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element ref="name"/>
+                <xs:element ref="value"/>
+            </xs:sequence>
+            <xs:attribute ref="os-release"/>
+            <xs:attribute ref="os-version"/>
+        </xs:complexType>
+    </xs:element>
+    <xs:element name="data">
+        <xs:complexType mixed="true">
+            <xs:sequence>
+                <xs:element ref="value"/>
+            </xs:sequence>
+            <xs:attribute ref="os-release"/>
+            <xs:attribute ref="os-version"/>
+            <xs:attribute ref="matchMode"/>
+            <xs:attribute ref="collectionSeparator"/>
+        </xs:complexType>
+    </xs:element>
+    <xs:element name="meta">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element ref="name"/>
+                <xs:element ref="value"/>
+            </xs:sequence>
+            <xs:attribute ref="os-release"/>
+            <xs:attribute ref="os-version"/>
+            <xs:attribute ref="matchMode"/>
+            <xs:attribute ref="collectionSeparator"/>
+        </xs:complexType>
+    </xs:element>
+    <xs:element name="nometa">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element ref="name"/>
+                <xs:element ref="value"/>
```

Comment:
Should we have a `value` child for `nometa`?  Could change in a follow-on PR if we don't want it